recipes/onnxruntime/all/conanfile.py

```diff
@@ -0,0 +1,348 @@
+
+        self._cmake.definitions["CMAKE_INSTALL_PREFIX"] = self.package_folder
+
+        if self.options.force_min_size_rel:
+            self._cmake.definitions["CMAKE_BUILD_TYPE"] = "MinSizeRel"
+        elif self.settings.build_type is not None:
+            build_type = str(self.settings.build_type)
+            self._cmake.definitions["CMAKE_BUILD_TYPE"] = build_type
+
+        if self.settings.os == "Android":
+            android_ndk_root = None
+            android_sdk_root = None
+            if "ANDROID_NDK_ROOT" in os.environ:
+                android_ndk_root = os.environ.get("ANDROID_NDK_ROOT")
+            elif "ANDROID_NDK_HOME" in os.environ:
+                android_ndk_root = os.environ.get("ANDROID_NDK_HOME")
+
+            if "ANDROID_SDK_ROOT" in os.environ:
+                android_sdk_root = os.environ.get("ANDROID_SDK_ROOT")
+            elif "ANDROID_SDK_HOME" in os.environ:
+                android_sdk_root = os.environ.get("ANDROID_SDK_HOME")
+            elif "ANDROID_HOME" in os.environ:
+                android_sdk_root = os.environ.get("ANDROID_HOME")
+
+            if android_ndk_root is None:
+                ndk_bundle_path = os.path.join(android_sdk_root, "ndk-bundle")
+                if os.path.isdir(ndk_bundle_path):
+                    android_ndk_root = ndk_bundle_path
+
+            if android_ndk_root is None:
+                raise Exception("ANDROID_NDK_ROOT env not defined")
+            if android_sdk_root is None:
+                raise Exception("ANDROID_SDK_ROOT env not defined")
+
+            self._cmake.definitions["CMAKE_TOOLCHAIN_FILE"] = os.path.join(
+                android_ndk_root, 'build', 'cmake', 'android.toolchain.cmake')
+            api_level = str(self.settings.os.api_level)
+            android_abi = tools.to_android_abi(self.settings.arch)
+            self._cmake.definitions["ANDROID_PLATFORM"] = \
+                f"android-{api_level}"
+            self._cmake.definitions["ANDROID_ABI"] = android_abi
+            self._cmake.definitions["ANDROID_MIN_SDK"] = api_level
+
+        self._cmake.definitions["ONNX_CUSTOM_PROTOC_EXECUTABLE"] = \
+            tools.which('protoc')
+
+        src_folder = os.path.join(self._source_subfolder, 'cmake')
+        self._cmake.configure(build_folder=self._build_subfolder,
+                              source_folder=src_folder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+
+        if self.options.shared:
+            self.copy(pattern="*.so", dst="lib", src=self._build_subfolder)
+            self.copy(pattern="*.dll", dst="lib", src=self._build_subfolder)
+        else:
+            self.copy(pattern="*.a", dst="lib", src=self._build_subfolder)
```

Comment:
- missing .lib files (both static and shared on windows)
- .so* in case onnxruntime create symbolic links with versions
- dll have to be packaged in bin directory

Reply:
Right, done